tools/ci_build/github/azure-pipelines/templates/set-test-data-variables-step.yml

```diff
@@ -1,7 +1,7 @@
 # sets variables $(TestDataUrl) and $(TestDataChecksum)
 
 parameters:
-  TestDataUrl: https://onnxruntimetestdata.blob.core.windows.net/models/20190327.zip
+  TestDataUrl: https://onnxruntimetestdata.blob.core.windows.net/models/20190419.zip
```

Comment:
For future reference - is there documentation on how to construct the zip with (potentially) any new models and upload it ?

Reply:
We use the same format as onnx model zoo. But, they don't have a doc.